rollup.config.ts

```diff
@@ -80,24 +79,6 @@ const nodePlugins = [
 export default (command: Record<string, unknown>): RollupOptions | RollupOptions[] => {
 	const { collectLicenses, writeLicense } = getLicenseHandler();
 	const commonJSBuild: RollupOptions = {
-		// fsevents is a dependency of chokidar that cannot be bundled as it contains binary code
-		external: [
-			'buffer',
-			'@rollup/plugin-typescript',
-			'assert',
-			'crypto',
-			'events',
-			'fs',
-			'fsevents',
-			'module',
-			'os',
-			'path',
-			'perf_hooks',
-			'process',
-			'stream',
-			'url',
-			'util'
-		],
 		input: {
 			'loadConfigFile.js': 'cli/run/loadConfigFile.ts',
 			'rollup.js': 'src/node-entry.ts'
```

Comment:
I was wondering why `fsevents` wasn't a problem but it turns out since you extracted the
```
const moduleName = 'fsevents'
```
constant in `fsevents-importer.ts`, the dynamic import can no longer be resolved by Rollup which is why it does not need to be marked as external. Which is fine. But this also means, we can simplify the `interop` option below to `interop: "defaultOnly"` and get rid of the function.

Reply:
"defaultOnly" is just a stricter version of "default", so if "defaultOnly" works, "default" works as well.

Reply:
> I was wondering why `fsevents` wasn't a problem but it turns out since you extracted the
> 
> ```
> const moduleName = 'fsevents'
> ```
> 
> constant in `fsevents-importer.ts`, the dynamic import can no longer be resolved by Rollup, which is why it does not need to be marked as external. Which is fine. 

I changed the dynamic import back to `fsevents`, but it's failing on `windows` and `linux` as it seems `rollup` is trying to load `fsevents` nonetheless, which fails [since it's also not installed]: https://github.com/rollup/rollup/runs/4876116960?check_suite_focus=true  is this a possible bug?

I'll change it back for the time being.

> But this also means, we can simplify the `interop` option below to `interop: "default"` and get rid of the function.

Done.

Reply:
> I changed the dynamic import back to fsevents, but it's failing on windows and linux

And did you also add "fsevents" back to the externals option? Because that is why it was here and why I originally though that externals should be an array containing just "fsevents".

Which I think is ok, i.e. to have your dependencies as "external". We could also employ a technique here used by many other plugins that just import their "package.json" file in their "rollup.config.js" and directly use package.dependencies as input for their "externals" option. That way it "auto-updates" should we ever have additional dependencies.

Reply:
Ah, this is the TypeScript plugin complaining, not Rollup. Of course you would need to teach TypeScript about the missing dependency. That was the reason why we had

```js

declare module 'fsevents' {
	export default {};
}
```

in  `declarations.dts`, but you did not like that in the past and removed it. You could just add it back :)

Reply:
Ok, I just checked one level of history and you are right 😉